runtime/doc/starting.txt

```diff
@@ -915,62 +914,59 @@ The viminfo file is used to store:
 
-The viminfo file is not supported when the |+viminfo| feature has been
-disabled at compile time.
-
-You could also use a Session file.  The difference is that the viminfo file
+You could also use a Session file.  The difference is that the ShaDa file
 does not depend on what you are working on.  There normally is only one
-viminfo file.  Session files are used to save the state of a specific editing
+ShaDa file.  Session files are used to save the state of a specific editing
 Session.  You could have several Session files, one for each project you are
-working on.  Viminfo and Session files together can be used to effectively
+working on.  ShaDa and Session files together can be used to effectively
 enter Vim and directly start working in your desired setup. |session-file|
 
-							*viminfo-read*
-When Vim is started and the 'viminfo' option is non-empty, the contents of
-the viminfo file are read and the info can be used in the appropriate places.
+							*shada-read*
+When Vim is started and the 'shada' option is non-empty, the contents of
+the ShaDa file are read and the info can be used in the appropriate places.
 The |v:oldfiles| variable is filled.  The marks are not read in at startup
-(but file marks are).  See |initialization| for how to set the 'viminfo'
+(but file marks are).  See |initialization| for how to set the 'shada'
 option upon startup.
 
-							*viminfo-write*
-When Vim exits and 'viminfo' is non-empty, the info is stored in the viminfo
-file (it's actually merged with the existing one, if one exists).  The
-'viminfo' option is a string containing information about what info should be
-stored, and contains limits on how much should be stored (see 'viminfo').
+							*shada-write*
+When Vim exits and 'shada' is non-empty, the info is stored in the ShaDa file 
+(it's actually merged with the existing one, if one exists |shada-merging|).  
+The 'shada' option is a string containing information about what info should 
+be stored, and contains limits on how much should be stored (see 'shada').
 
 Notes for Unix:
-- The file protection for the viminfo file will be set to prevent other users
+- The file protection for the ShaDa file will be set to prevent other users
   from being able to read it, because it may contain any text or commands that
   you have worked with.
-- If you want to share the viminfo file with other users (e.g. when you "su"
+- If you want to share the ShaDa file with other users (e.g. when you "su"
   to another user), you can make the file writable for the group or everybody.
-  Vim will preserve this when writing new viminfo files.  Be careful, don't
-  allow just anybody to read and write your viminfo file!
-- Vim will not overwrite a viminfo file that is not writable by the current
+  Vim will preserve this when writing new ShaDa files.  Be careful, don't
+  allow just anybody to read and write your ShaDa file!
+- Vim will not overwrite a ShaDa file that is not writable by the current
```

Comment:
> I tried again with e287807, but I still see no E137 when using vimrc. I see the error for read only file main.shada when using nvim -u NONE and exiting or using :wsh.

I see this error even when using

    bin/nvim -u NONE --cmd 'set nomore shortmess=filmnrwxaoOstTWAlc noshowcmd cmdheight=10' -c qa

(used every advice from :h hit-enter which is supposed to allow you to avoid some hit-enter prompts). Using your vimrc makes no difference. I have only it, default plugins and plugins listed in that vimrc installed.

I would highly suggest to reduce the vimrc. YCM is rather bold and configurable. I installed it from scratch with `./install --clang-completer --system-libclang --system-boost`.

>
> —
> Reply to this email directly or view it on GitHub.